### ISSUE TITLE
Update aegit_Template version to 2.0.0

### ISSUE THREAD

**Issue by peterennis** [peterennis/aegit_Template#17]
@kinsidar, Make it match with aegit.
Keep the file naming similar.

Ref: https://github.com/peterennis/aegit/blob/master/zip/adaept%20revision%20control%20accdb%20v200%2020171010.zip

Compare with:
https://github.com/peterennis/aegit_Template/blob/master/zip/aegit_Template_v158.zip

Update this ticket with your suggested file name before making a new PR.

**Comment by kinsidar** [comment]
I agree with you on making the aegit_Template version same as aegit in order to reduce confusion.

I'll update the name to 
> adaept revision control template accdb v200 20171010
